api/integrations/lead_tracking/hubspot/client.py

```diff
@@ -42,6 +49,51 @@ def get_contact(self, user: FFAdminUser) -> None | dict:
 
         return results[0]
 
+    def create_lead_form(self, user: FFAdminUser, hubspot_cookie: str) -> dict:
```

Comment:
nit: we should avoid empty `dict` annotations. Depending on our needs, we should either come up with a `TypedDict` or use `dict[str, Any]`.

Reply:
Good idea. Implemented.